generator-jhipster-gomicro/generators/server/templates/go/go/handler/db.go

```diff
@@ -0,0 +1,314 @@
+package handler
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+	"github.com/google/uuid"
+	"github.com/micro/micro/v3/service/errors"
+	"github.com/micro/micro/v3/service/logger"
+	db "<%= packageName %>/proto"
+	gorm2 "<%= packageName %>/pkg"
+	"google.golang.org/protobuf/types/known/structpb"
+	"gorm.io/datatypes"
+	<%_ if (postgresql){  _%>
+	"gorm.io/gorm"
+	"gorm.io/driver/postgres"
+	<%_ } _%>
+    <%_ if (mongodb){ _%> 
+	"go.mongodb.org/mongo-driver/mongo"
+	"log"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	<%_ } _%>
+)
+const idKey = "id"
+const _idKey = "_id"
+<%_ if (postgresql){  _%>
+const stmt = "create table if not exists %v(id text not null, data jsonb, primary key(id)); alter table %v add created_at timestamptz; alter table %v add updated_at timestamptz"
+<%_ } _%>
+
+type Record struct {
+	ID   string  `json:"_id" bson:"_id" `
+	Data datatypes.JSON `json:"data"`
+	// private field, ignored from gorm
+	table     string `gorm:"-"`
+	CreatedAt time.Time
+	UpdatedAt time.Time
+}
+
+type Db struct {
+	gorm2.Helper
+}
+
+<%_ if (postgresql){  _%>
+func (e *Db) Create(ctx context.Context, req *db.CreateRequest, rsp *db.CreateResponse) error {
+	if len(req.Record.AsMap()) == 0 {
+		return errors.BadRequest("db.create", "missing record")
+	}
+	tableName :="temp"
+	logger.Infof("Inserting into table '%v'", tableName)
+	db, err := gorm.Open(postgres.Open("postgresql://go@localhost:5433/postgres"), &gorm.Config{})   
+	if err != nil {
+		logger.Infof("hii error")
+		return err
+	}
+	db.Exec(fmt.Sprintf(stmt, tableName, tableName, tableName))
+	m := req.Record.AsMap()
+	id := req.Id
+	if len(id) == 0 {
+		if mid, ok := m[idKey].(string); ok {
+			id = mid
+		} else {
+			id = uuid.New().String()
+			m[idKey] = id
+		}
+	}
+	bs, _ := json.Marshal(m)
+	err = db.Table(tableName).Create(&Record{
+		ID:   id,
+		Data: bs,
+	}).Error
+	if err != nil {
+		return err
+	}
+
+	// set the response id
+	rsp.Id = id
+	logger.Infof("hii error5")
+
+	return nil
+}
+
+func (e *Db) Update(ctx context.Context, req *db.UpdateRequest, rsp *db.UpdateResponse) error {
+	if len(req.Record.AsMap()) == 0 {
+		return errors.BadRequest("db.update", "missing record")
+	}
+	tableName :="temp"
+	logger.Infof("Updating table '%v'", tableName)
+	db, err := gorm.Open(postgres.Open("postgresql://go@localhost:5433/postgres"), &gorm.Config{})   
```

Comment:
this value as to be picked from ENV 
DB_URL=postgresql://go@localhost:5433/postgres